reddit.js

```diff
@@ -2,4 +2,43 @@ var reddit = require("./data.json")
 
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+//forEach iterator for all titles
+
+reddit["data"].children.forEach(function (title) {
+	console.log(title.data.title);
+});
+
+//map iterator for array of permalinks
+
+var permalinks = reddit["data"].children.map(function (link) {
```

Comment:
Same here as with the above, I would use something like ```post``` or ```child``` instead of ```link``` as a parameter.